bindings/cs/MongoDB.Libmongocrypt.Test/Callbacks/HashCallbackTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * Copyright 2020–present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using FluentAssertions;
+using MongoDB.Libmongocrypt.Callbacks;
+using Xunit;
+
+namespace MongoDB.Libmongocrypt.Test.Callbacks
+{
+    public class HashCallbackTests
+    {
+        [Fact]
+        public void HashTest()
+        {
```

Comment:
test data can be found here https://docs.google.com/document/d/1-qmr6miUug2qciOlX2Q7eisChvG_RtzLNJnp6ZtmtvY/edit?ts=5f93353d#heading=h.mp30qfn97460

Reply:
Answered above